channels/us.m3u

```diff
@@ -6293,9 +6293,9 @@ https://venntv-rakuten-samsung.amagi.tv/playlist.m3u8
 https://venntv-roku.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Entertainment",VENN (1080p)
 https://venntv-samsungus.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="Venn.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Entertainment",VENN
+#EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Entertainment",VENN
 https://venntv-vizio.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="VennTVEngland.us" tvg-name="VENN (England)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Sports",VENN (England)
+#EXTINF:-1 tvg-id="VENNEngland.UK" tvg-name="VENN (England)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Sports",VENN (England)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN (England)" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/zVoX1fM.jpg" group-title="Sports",VENN (England)
```

This channel is American by nature, streamed for the English people in the UK.

Reply:
Okay, so keep .us?

Reply:
If you're sure it's intended for the UK both sources should be changed identically

Reply:
Did you change that, I think I might have replaced it

Reply:
Just copy line 6296 into 6298 and I'm approving it

Reply:
Okay I think I have it per your suggestions

Reply:
> 
> 
> If you're sure it's intended for the UK both sources should be changed identically

It's for our case only. As I said, VENN is from America. The link indicates "samsunguk", so I suppose it's streamed for the UK people. It works the same as Pluto TV.

Reply:
the channel itself is for everyone and people behind it are in LA. This is just a CDN